client/my-sites/exporter/post-type-options.jsx

```diff
@@ -69,28 +74,39 @@ const PostTypeOptions = React.createClass( {
 	},
 
 	renderFields() {
-		const { options } = this.props;
+		const {
+			options,
+			siteId,
+			postType,
+			values,
+		} = this.props;
 
 		const Field = ( props ) => {
```

Comment:
Hmm, it was the simplest and most DRY way I could think of to reuse a lot of the props in the PostTypeOptions component without explicitly providing them on each usage of `Field` below. Open to other suggestions :)